src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -2623,10 +2623,31 @@ struct R { }
             var type = compilation.GlobalNamespace.GetMember<NamedTypeSymbol>("C");
             Assert.False(type.GetMember<FieldSymbol>("f1").Type.IsManagedType);
             Assert.False(type.GetMember<FieldSymbol>("f2").Type.IsManagedType);
-            Assert.True(type.GetMember<FieldSymbol>("f3").Type.IsManagedType);
+            Assert.True(type.GetMember<FieldSymbol>("f3").Type.IsManagedType); // is managed due to being ErrorType due to protection level (CS0169)
             Assert.True(type.GetMember<FieldSymbol>("f4").Type.IsManagedType);
         }
 
+        [Fact]
+        public void IsManagedType_GenericStruct()
```

Comment:
Should have a test when the generic argument is an error type. Example `S<Widget>` won't be able to bind the type `Widget`.

Reply:
Done